Mumbere7/quickCredit
# #165946914 fix-endpoints-erros

#### What does this PR do?
This pull request is fixing errors related to the all the endpoints.
#### Description of Task to be completed?
Have the following working endpoints:
GET api/v1/users
POST api/v1/users
GET api/v1/users/:id
PUT api/v1/users/:email/verify
GET api/v1/loans
GET api/v1/loans:id/
POST api/v1/loans/apply
PATCH api/v1/loans/:id/
#### How should this be manually tested?
To test this endpoints a user should clone this project and 
in the terminal run  => npm install 
#### Any background context you want to provide?
Not at the moment.
#### What are the relevant pivotal tracker stories?
#165946914
#### Screenshots (if appropriate)
#### Questions:
